### ISSUE TITLE
Weird behaviour generating documentation comments for internal procedures

### ISSUE THREAD

**Issue by rvanbekkum** [365businessdev/vscode-alxmldocumentation#10]
Dear @365businessdev,
I have ran into some weird issues trying to add documentation comments to internal procedures.

1. It seems to be doing something wrong if there is a local procedure beneath an internal procedure. I think this is best explained with an example:
    ```pascal
    internal procedure MyInternalProcedure(SomeInterface: Interface "Line With Price"; SomeRec: Record "Custom Address Format"): Boolean
    var
        SomeCodeunit: Codeunit "Sales-Post";
    begin
        exit(true);
    end;

    local procedure MyLocalProcedure(): Integer
    begin
        exit(42);
    end;
    ```
    Putting the code above in a codeunit and then typing `///` above the internal procedure, results in:
    ![image](https://user-images.githubusercontent.com/7383241/86898890-8e085a80-c109-11ea-9ad3-215e01af7b52.png)

2. If I have an internal procedure at the bottom of a codeunit, typing `///` doesn't trigger any documentation comment generation.

**Comment by sirhc101** [comment]
You're absolutly right! `internal procedure` is not working properly. Fixed this in `dev` branch already and preparing for release

**Comment by sirhc101** [comment]
Fixed in version 0.3.1
